Dockerfile

```diff
@@ -31,9 +31,11 @@ ENV DEVKITPPC=/devkitpro/devkitPPC
 # Now we have a container that has the dev environment set up. 
 # Copy current folder into container, then compile
 COPY . /projectroot/
-RUN cd /projectroot && make
+RUN cd /projectroot && make clean && make -j8 package
 
 
 # Copy the DOL and ELF out of the container
 FROM scratch AS export-stage
-COPY --from=usbloader /projectroot/boot.* /
+COPY --from=usbloader /projectroot/boot.* / 
+COPY --from=usbloader /projectroot/usbloader_gx /
+COPY --from=usbloader /projectroot/usbloader_gx.zip /
```

Comment:
Since I have the build running the packaging now, we need to copy out the usbloader_gx package